src/AutorestSwift/View Models/ConstantViewModel.swift

```diff
@@ -39,10 +39,10 @@ struct ConstantViewModel {
 
     /// Initialize from Value type (such as Property or Parameter)
     init(from schema: ConstantSchema) {
-        let name = (schema.serializedName ?? schema.name).lowercasedFirst
+        let name = schema.name.lowercasedFirst
         assert(!name.isEmpty)
         self.name = name
-        self.serializedName = (schema.serializedName ?? schema.name)
+        self.serializedName = schema.serializedName ?? schema.name
```

Comment:
@daviwil is this a good assumption? If serialized name is omitted, I suppose it would make sense to use the regular name to send it over the wire.